db-engine/src/main/java/com/splicemachine/db/iapi/sql/dictionary/StatementColumnPermission.java

```diff
@@ -115,7 +115,7 @@ public void check( LanguageConnectionContext lcc,
 
         String currentUserId = lcc.getCurrentUserId(activation);
 
-		FormatableBitSet permittedColumns = null;
+			FormatableBitSet permittedColumns = null;
```

Comment:
When populating the permittedColumns, in addition to the call of addPermittedColumns() for public and currentUserId, I think we should add the call for currentGroupUser too.